pkg/processor/trigger/kafka/trigger.go

```diff
@@ -272,6 +272,12 @@ func (k *kafka) ConsumeClaim(session sarama.ConsumerGroupSession, claim sarama.C
 			// trigger is ready for rebalance if both the handler is done and
 			// the workers are finished draining events
 			go func() {
+				defer func() {
+					if err := recover(); err != nil {
```

Comment:
You can use our `common.CatchAndLogPanicWithOptions`, which logs the panic in a more organized fashion

Reply:
@TomerShor Great idea! will do